libs/designsystem/modal/v2/src/modal/modal.component.spec.ts

```diff
@@ -7,13 +7,14 @@ import {
   IonModal,
   IonTitle,
   IonToolbar,
-} from '@ionic/angular';
+} from '@ionic/angular/standalone';
 import { MockComponents } from 'ng-mocks';
 import { IconComponent } from '@kirbydesign/designsystem/icon';
 import { ButtonComponent } from '@kirbydesign/designsystem/button';
 import { ModalV2Component } from './modal.component';
 
-describe('FullscreenComponent', () => {
+// TODO(@RasmusKjeldgaard): Fix clean up in this test.
```

Comment:
I've played around with this a bit but cannot make it work. We are removing this experimental component very soon anyways, so you can just go ahead and remove the entire spec.